### PR TITLE
LPS-188034 Make solution compatible with our CI, which runs Chromium 100

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hey,

I need to resend this pull request so CI accepts it.

As of today our CI runs Chromium 100 which is not compatible with `checkVisibility` what is only compatible from Chromium 105 on.

### CODE REVIEW

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js [hunk 1]
```diff
@@ -454,10 +454,20 @@ AUI.add(
 
 					const contentBox = instance._contentBox;
 
+					let computedStyle;
+
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js:458]: Is there a reason to have this variable outside of local scope? If no, it's a better practice to keep it in. Something like:
```js
const computedStyle = window.getComputedStyle(node);
```

**Review by markocikos — COMMENTED** [review]
@antonio-ortega see question inline. Other than that, looks good!

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js (hunk 1 above)
Comment (antonio-ortega) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js]: Not really. Thanks for the suggestion, @markocikos !

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

**Review by markocikos — APPROVED** [review]
@antonio-ortega LGTM! :sparkles:

**Comment by markocikos** [comment]
ci:forward

**Comment by deborasoliveira** [comment]
I'll take a look at the relevant result.

**Comment by deborasoliveira** [comment]
Hi @antonio-ortega! _semantic-versioning-jdk8/0/0_ is failing in the relevant run. Could you take a look at this?
```
jar:
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/portal-impl/classes/META-INF/maven/com.liferay.portal/com.liferay.portal.impl
[propertyfile] Creating new property file: /opt/dev/projects/github/liferay-portal/portal-impl/classes/META-INF/maven/com.liferay.portal/com.liferay.portal.impl/pom.properties
[bnd-invoker] SLF4J: Class path contains multiple SLF4J bindings.
[bnd-invoker] SLF4J: Found binding in [jar:file:/tmp/4805143793594666153.tmp!/org/slf4j/impl/StaticLoggerBinder.class]
[bnd-invoker] SLF4J: Found binding in [jar:file:/opt/dev/projects/github/liferay-jenkins-ee/commands/../lib/jenkins/slf4j-log4j12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[bnd-invoker] SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[bnd-invoker] SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
[bnd-invoker] 2 WARNINGS
[bnd-invoker]  Export com.liferay.portal.deploy.hot,  has 1,  private references [com.liferay.portal.spring.aop], 
[bnd-invoker]  Export com.liferay.portal.servlet,  has 1,  private references [com.liferay.portal.struts], 
 [sync-dir] Synchronizing /opt/dev/projects/github/liferay-binaries-cache-2020/.gradle/caches/modules-2/files-2.1 into /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1
 [sync-dir] 0 files synchronized in 416ms
     [echo] Executing Gradle task: baseline
     [exec] Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
     [exec] To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/6.9.2/userguide/gradle_daemon.html#sec:disabling_the_daemon.
     [exec] Daemon will be stopped at the end of the build 
     [exec] > Task :classes UP-TO-DATE
     [exec] > Task :jar SKIPPED
     [exec] 
     [exec] > Task :baseline
     [exec] [Baseline Warning] Bundle Version Change Recommended: 82.0.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/portal-impl/portal-impl.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.impl/81.0.1/a9a5e9fdc6a6e2298a23bf7676ce3f14bb954186/com.liferay.portal.impl-81.0.1.jar
     [exec] 
     [exec] Gradle build finished at 2023-07-03 11:13:09.540.
     [exec] 
     [exec] BUILD SUCCESSFUL in 8s
     [exec] 1 actionable task: 1 executed
     [exec] 
     [exec] See the profiling report at: file:///opt/dev/projects/github/liferay-portal/portal-impl/build/reports/profile/profile-2023-07-03-04-13-02.html
     [exec] A fine-grained performance profile is available: use the --scan option.

init-deploy: 
```

**Comment by antonio-ortega** [comment]
Thanks @deborasoliveira ,

Although I don't think it's strictly related to my changes, there's been some change in that versioning since I sent the pull request so I've rebased and resent it.

Let's hope it works now 🤞

**Comment by antonio-ortega** [comment]
ci:test:sf

**Comment by antonio-ortega** [comment]
ci:test:relevant

**Comment by antonio-ortega** [comment]
Hey @deborasoliveira ,

Could you check now whether these errors are related, please?

Thanks in advance!

**Comment by deborasoliveira** [comment]
Hi @antonio-ortega! I started the analysis but I'll finish it tomorrow.

**Comment by markocikos** [comment]
ci:forward

**Comment by deborasoliveira** [comment]
Hi @antonio-ortega! The failures are not related to the PR!
Some of the failed tests were executed locally using the PR bundle and passed, and the other failures are flaky. 

[Comparison run](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=2&testrayRunIdB=2370224774&testrayRunIdA=2369774879&statusB=3)

**Comment by antonio-ortega** [comment]
ci:forward:force

**Comment by antonio-ortega** [comment]
Manually forwarded https://github.com/brianchandotcom/liferay-portal/pull/137466

**Comment by bryceosterhaus** [comment]
@antonio-ortega any idea on the effort to get CI running on a more recent version on chromium? Or do you know if the CI infra team is going to update at all? I hate to update our code just to support on older version of a browser that we don't support for customers.

**Comment by antonio-ortega** [comment]
Hey @bryceosterhaus ,

Totally agree. I asked about it, and it seems a bit more tricky than expected. You can track the progress [here](https://liferay.atlassian.net/browse/LRCI-1855)

**Comment by bryceosterhaus** [comment]
@antonio-ortega awesome, so long as there is momentum to move forward that is good for me. Thanks!

**Pull request closed without merging**